zha/application/platforms/light/__init__.py

```diff
@@ -1128,7 +1128,29 @@ class LightGroup(GroupEntity, BaseLight):
 
     def __init__(self, group: Group):
         """Initialize a light group."""
-        super().__init__(group)
+        # light groups change the update_group_from_child_delay so we need to do this
+        # before calling super
+        light_options = group.gateway.config.config.light_options
+        self._zha_config_transition = light_options.default_light_transition
+        self._zha_config_enable_light_transitioning_flag = (
+            light_options.enable_light_transitioning_flag
+        )
+        self._zha_config_always_prefer_xy_color_mode = (
+            light_options.always_prefer_xy_color_mode
+        )
+        self._zha_config_group_members_assume_state = (
+            light_options.group_members_assume_state
+        )
+        if self._zha_config_group_members_assume_state:
+            update_group_from_member_delay = ASSUME_UPDATE_GROUP_FROM_CHILD_DELAY
+        else:
+            update_group_from_member_delay = 0.5
```

Comment:
The non-assume delay of 0.5 seconds is defined in two places now (here and in `GroupEntity`).
IMO ideally, it would only be in one place: `GroupEntity`.

And/or maybe use a constant for it too, so the meaning is clearer? We had `DEFAULT_UPDATE_GROUP_FROM_CHILD_DELAY` before.

Reply:
updated